tests/Test.ADAL.NET.Unit.net45/InteractiveAuthTests.cs

```diff
@@ -169,20 +170,31 @@ public async Task ExtendedLifetimeRetryAsync()
         public async Task AcquireTokenPositiveWithoutUserIdAsync()
         {
             _context = new AuthenticationContext(AdalTestConstants.DefaultAuthorityCommonTenant, new TokenCache());
-            MockHelpers.ConfigureMockWebUI(new AuthorizationResult(AuthorizationStatus.Success,
-                AdalTestConstants.DefaultRedirectUri + "?code=some-code"));
-            AdalHttpMessageHandlerFactory.AddMockHandler(new MockHttpMessageHandler(AdalTestConstants.GetTokenEndpoint(AdalTestConstants.DefaultAuthorityCommonTenant))
+            var mockWebUI = MockHelpers.ConfigureMockWebUI(
+                new AuthorizationResult(
+                    AuthorizationStatus.Success,
```

Comment:
There is an extra Integration test that needs to be written - what happens if the PKCE code does not match the code chanllenge. I plan to do this test when I expose the ICustomWebUI interface.